sunpy/io/tests/test_ana.py

```diff
@@ -75,3 +75,12 @@ def test_f32c():
     afilename = tempfile.NamedTemporaryFile().name
     with pytest.raises(RuntimeError):
         ana.write(afilename, img_f32, 'testcase', 1)
+
+
+@skip_ana
+def test_read_memmap():
+    # Test to check memmap is passed to the reader
+    afilename = tempfile.NamedTemporaryFile().name
+    ana.write(afilename, img_f32, 'testcase', 0)
+    img_f32c_rec = ana.read(afilename, memmap=True)
+    assert np.sum(img_f32c_rec[0][0] - img_f32) == 0
```

Comment:
Can we check that this array is not memory mapped?

Reply:
One question arises, Do we need memmap testing for `ana` and `jp2` readers if only the FITS reader supports it?

Reply:
We should at at least check the keyword doesn't error and the array is not a memory map.

Reply:
I added the following, is it correct?

Reply:
Yeah